java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -266,6 +269,111 @@ CompletableFuture<ClusterValue<Map<String, Object>[]>> functionList(
      */
     CompletableFuture<String> functionDelete(String libName, Route route);
 
+    /**
+     * Returns the serialized payload of all loaded libraries.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-dump/">redis.io</a> for details.
+     * @return The serialized payload.
```

Comment:
might be slightly clearer although I know we already stated this above, up to you. Same applies elsewhere in this PRs docs
```suggestion
     * @return The serialized payload of all loaded libraries.
```